man/man8/zpool-initialize.8

```diff
@@ -71,6 +72,10 @@ with no flags can be used to re-initialize all unallocated regions on
 the relevant target devices.
 .It Fl w , -wait
 Wait until the devices have finished initializing before returning.
+.It Fl a , -all
+Begin, cancel, suspend initializing on
+all
+pools.
```

Comment:
Here and in other man pages, while existing parameters are not really alphabetically sorted, I am not sure `-a` has any reason to be the last.